template-sample-app/template/src/components/common/Map.js

```diff
@@ -10,6 +10,20 @@ import { setViewState } from '@carto/react/redux';
 import { BASEMAPS, GoogleMap } from '@carto/react/basemaps';
 
 const useStyles = makeStyles((theme) => ({
+  root: {
```

Comment:
A quite curious change. It looks good, but I'm not sure this marging should be added here, inside the component (where it's more difficult to remove). I would probably had chosen adding this externally, around the <Map> component, in a kind of small MapPanel wrapper.... anyway, I have tested it locally and it looks good visually

Reply:
I've added this here because Main is already has enough complexity and this visual changes has only to do with the map. As this component is from the template I think that this is the best place to style it.